File: globalGatheringMobile/global_gathering_application_1/lib/model/reponse/post_response.dart
class PostResponse {
  String? uuid;
  String? comment;
  String? relatedEvent;
  String? uri;
  String? createdBy;
  String? createdAt;

  PostResponse(
      {this.uuid,
      this.comment,
      this.relatedEvent,
      this.uri,
      this.createdBy,
      this.createdAt});

  PostResponse.fromJson(Map<String, dynamic> json) {
    uuid = json['uuid'];
    comment = json['comment'];
    relatedEvent = json['relatedEvent'];
    uri = json['uri'];
    createdBy = json['createdBy'];
    createdAt = json['createdAt'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['uuid'] = this.uuid;
    data['comment'] = this.comment;
    data['relatedEvent'] = this.relatedEvent;
    data['uri'] = this.uri;
    data['createdBy'] = this.createdBy;
    data['createdAt'] = this.createdAt;
    return data;
  }
}

File: globalGatheringAPI/globalgathering/src/main/java/com/salesianos/triana/edu/globalgathering/dto/post/PostResponse.java
package com.salesianos.triana.edu.globalgathering.dto.post;

import com.salesianos.triana.edu.globalgathering.model.Post;
import com.salesianos.triana.edu.globalgathering.service.post.MediaTypeUrlResource;

import jakarta.persistence.OneToOne;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.CreatedDate;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PostResponse {

    private String name;
    private String uri;
    private String type;
    private String createdBy;
    @CreatedDate
    private LocalDateTime createdAt;
    private long size;


}
